cli/cmd/alert_channel.go

```diff
@@ -0,0 +1,220 @@
+package cmd
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/api"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	alertChannelCommand = &cobra.Command{
+		Use:     "alert-channel",
+		Aliases: []string{"alert-channels", "ac"},
+		Short:   "Manage alert channels",
+		Long:    "Manage alert channels integrations with Lacework",
+	}
+
+	// alertChannelsListCmd represents the list sub-command inside the alert channels command
+	alertChannelListCmd = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all available alert channel integrations",
+		Args:    cobra.NoArgs,
+		RunE:    alertChannelList,
+	}
+
+	// alertChannelShowCmd represents the show sub-command inside the alert channel command
+	alertChannelShowCmd = &cobra.Command{
+		Use:     "show",
+		Aliases: []string{"get"},
+		Short:   "Show a single alert channel integration",
+		Args:    cobra.ExactArgs(1),
+		RunE:    alertChannelShow,
```

Comment:
Some of the fields from the show command are truncated.... Can we fix that?

<img width="962" alt="Screen Shot 2022-09-07 at 12 27 30 AM" src="https://user-images.githubusercontent.com/5712253/188816031-c2dc602e-4f58-4d5b-a135-9c2974c2dd4b.png">

Reply:
This was intentional as some of the longer values messed with table format. I will remove it though.